src/components/DateSeparator/DateSeparatorStyles.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import { device } from '@/constants/mediaQueries';
+
+export const SeparatorBoxStyled = styled.div`
+  display: flex;
+  align-items: center;
+  margin: 16px 0;
+  color: var(--color-grey-9);
+  @media ${device.tablet} {
+    margin: 24px 0;
+  }
+`;
+
+export const Line1 = styled.div`
```

Comment:
Is it possible to do it with pseudoelements :after, :before. I'm not sure it`s better))

Reply:
with pseudoelements I had problems with styles, so I took the simpler route))